src/main/java/com/redhat/developers/msa/hello/ZipkinResource.java

```diff
@@ -32,8 +32,12 @@
     @Produces
     @Singleton
     public Brave getBrave() {
+    	String zipkinUrl = System.getenv("ZIPKIN_SERVER_URL");
```

Comment:
Ah - thought I had reverted all the defaults back to zipkin-query :)
I'll place the default in the Dockerfile - so I guess the fabric8 maven plugin uses this file aswell?